package.json

```diff
@@ -43,7 +43,8 @@
     "build:watch": "npm run build -- --watch",
     "prepublish": "npm run build"
   },
-  "main": "dist/apollo-upload-client.js",
+  "main": "src/index.js",
```

Comment:
My bad, I was testing it without transpilation, and forget to remove it after done.